examples/SDO_to_STEREO_Coordinate_Conversion.py

```diff
@@ -0,0 +1,149 @@
+"""
+===================================
+AIA to STEREO Coordinate Conversion
+===================================
+
+In this example we demonstrate how you can identify a point or region on the
+surface of the Sun in an AIA image and then convert that point to a point in a
+STEREO image.
+
+.. note::
+    This example requires `WCSAxes` which is an optional SunPy dependency.
+"""
+
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import wcsaxes
+
+import sunpy.map
+import sunpy.coordinates
+import sunpy.coordinates.wcs_utils
+from sunpy.net import vso
+
+
+###############################################################################
+# The first step is to download some data, we are going to get an image from
+# early 2011 when the STEREO spacecraft were roughly 90 deg seperated from the
+# Earth.
+
+stereo = (vso.attrs.Source('STEREO_B') &
+          vso.attrs.Instrument('EUVI') &
+          vso.attrs.Time('2011/1/1', '2011/1/1T00:10:00'))
```

Comment:
I know we can handle this date format: `'2011/1/1T00:10:00'` but I keep preferring, if using `T` using the iso format way, so no confusion between month and day, ie. `2011-01-01T00:10:00`
But this is me maybe being picky...